packages/react-api/src/Api.tsx

```diff
@@ -280,6 +281,21 @@ export function ApiCtxRoot ({ apiUrl, children, isElectron, store }: Props): Rea
     [apiError, createLink, extensions, isApiConnected, isApiInitialized, isElectron, state, apiEndpoint, apiRelay, apiUrl]
   );
 
+  useEffect((): void => {
+    if (wallet.account) {
```

Comment:
```suggestion
    if (!wallet.account) return
```

return early @KarishmaBothara, don't forget 😉